forte/data/data_store.py

```diff
@@ -183,10 +182,25 @@ def __init__(self, onto_file_path: Optional[str] = None):
     def _new_tid(self) -> int:
         r"""This function generates a new ``tid`` for an entry."""
         return uuid.uuid4().int
+    
+    def _add_new_type(self, type_name: str):
+        """
+        Add a new type input_entry_class into self._type_attributes.
+        This method will call get_entry_attribute_by_class(class)
+        Args:
+        input_entry_class: A string or class type representing a class
+        """
+        # check if type is in dictionary
+        if type_name in self._type_attributes:
+            return
+        # get attribute dictionary
+        self._type_attributes[type_name] = {"attributes":{}, "parent": None}
```

Comment:
If I didn't misunderstand, this behavior is not correct.

If the `type_name` is not in `self._type_attributes`, we should check to see if we can import the specific class. Now it looks like it is just accepting a new type name, and assigning empty attributes.

Reply:
Also, I think the function should also be designed to return the content from `self._type_attributes`, so you won't need to access it again (Like in line 252). And also use the EAFP pattern here.